### PR TITLE
removed .gradle from file

### PR DESCRIPTION
**Pull request by bilkis96**

### CODE REVIEW

**Comment by neon-lab-dev** [comment]
@bilkis96 please check the review points added.

cc- @ritik-agrawal

File: src/main/java/com/neonlab/loginservice/util/DateUtils.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.neonlab.loginservice.util;
+
+import java.util.Date;
+
```
Comment (neon-lab-dev) [inline src/main/java/com/neonlab/loginservice/util/DateUtils.java]: Use @UtilityClass annotation

cc- @bilkis96 @ritik-agrawal

File: src/main/java/com/neonlab/loginservice/util/UUIDEncryptor.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import javax.crypto.spec.SecretKeySpec;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class UUIDEncryptor {
+
+    public static String encryptUUID(String uuid) throws Exception {
+        // Create AES cipher
+        Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5Padding");
+        SecretKeySpec secretKeySpec = new SecretKeySpec(generateAESKey().getBytes(), "AES");
```
Comment (neon-lab-dev) [inline src/main/java/com/neonlab/loginservice/util/UUIDEncryptor.java]: Move "AES" to constants class

cc- @bilkis96 @ritik-agrawal

File: src/main/java/com/neonlab/loginservice/util/UUIDEncryptor.java
Comment (neon-lab-dev) [inline src/main/java/com/neonlab/loginservice/util/UUIDEncryptor.java:1]: Move this class to Common

cc- @bilkis96 @ritik-agrawal

File: src/main/java/com/neonlab/loginservice/config/Constants.java
[the remark this answers is not in the document]
Comment (bilkis96) [reply src/main/java/com/neonlab/loginservice/config/Constants.java:1]: move it to common

**Pull request closed without merging**